domains/benji.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "benjitusk",
+        "email": "benjitusk1@gmail.com"
+    },
+    "record": {
+        "CNAME": "google.com"
```

Comment:
You can't point your domain to google

Reply:
Yeah, I was checking out the website and wanted to see the registration process but didn't have a website set up yet.